src/nrfx/hal/nrf_gpio.h

```diff
@@ -0,0 +1 @@
+#include "../drivers/nrfx_common.h"
```

Comment:
Please add a ifndef guard already in these 3 ones (I know it is not needed yet, but it will avoid somebody forgetting it later